frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -106,3 +106,12 @@ body {
     background-color: transparent;
   }
 }
+
+/* this is to make it work when exporting to pdfs,
+   where the EmbedFrame is not part of the exported dom */
+.WithSdkContentContainer {
```

Comment:
Is the only diff from `WithThemeBackground` the second parameter `-mb-color-background`?
If so, we we add the fallback value directly on `WithThemeBackground` so we have one less case to think about/troubleshoot?
Debugging the pdf export is tricky in part because we have so many variants and we're adding code (variants) to fix it which doesn't seem very good in the long term

Reply:
@npretto good point, the problem is that we only apply `WithThemeBackground` this in `[data-metabase-night]` though - that is why I created a new `WithSdkContentContainer` class, so it is applied regardless of the `[data-metabase]` key.

Let me check with @WiNloSt if we can move `.WithThemeBackground` out of the `[data-metabase-theme]` block.

```ts
[data-metabase-theme="night"][data-metabase-theme="night"] {
  --mb-color-text-selected: var(--mb-base-color-white);
  --mb-color-text-hover: var(--mb-base-color-brand-30);
  --mb-color-border: var(--mb-color-text-medium);
  --mb-color-bg-dashboard: var(--mb-color-bg-black);
  --mb-color-background: var(--mb-color-bg-black);
  --mb-color-background-selected: var(--mb-base-color-brand-60);
  --mb-color-background-hover: var(--mb-base-color-brand-70);
  --mb-color-background-disabled: var(--mb-base-color-gray-70);
  --mb-color-background-brand: var(--mb-base-color-brand-60);

  .EmbedFrame,
    /* this is to make it work when exporting to pdfs,
  where the EmbedFrame is not part of the exported dom */
      .WithThemeBackground {
    /* the dashboard background color is not set by default in the embedding sdk */
    background-color: var(--mb-color-bg-dashboard, --mb-color-background);
    border-color: var(--mb-color-border);
  }
```